src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -418,6 +418,10 @@ module.exports = class LevelPlane {
     let pos = [];
     let offset = [];
     let pistonType = this.getBlockAt(position).blockType;
+    let checkSticky = pistonType.substring(pistonType.length - 6, pistonType.length);
+    if (checkSticky === "Sticky") {
```

Comment:
this (and the several instances below) should be in a `levelBlock.getIsStickyPiston()` method